tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -16,9 +16,8 @@ class CompletionArgSuite extends BaseCompletionSuite {
         |""".stripMargin,
     """|assertion = : Boolean
        |Main arg
-       |:: scala.collection.immutable
        |""".stripMargin,
-    topLines = Option(3)
+    topLines = Option(2)
```

Comment:
Why change this?

Reply:
after changing the ordering for scala 2 - `name.decoded` the third symbol come either `::` or `#::` - depends on scala version and that's correct due to collections rework.

I decided that decreasing the number here is better than adding compat section because other values aren't about `ArgSuite`

Reply:
Ok 👌